### ISSUE TITLE
Question about training and dataset for segmentation

### ISSUE THREAD

**Issue by akatav** [ckjoung/ecg-cgseg#1]
Thank you for sharing your code with us. We have a few questions about the code  and hope it is ok to raise them as 'issues' here instead of  emailing back and forth. Do let me know if it is  ok. 

1. The train_model() in train.py takes the first 100 files in LUDB for training and the next 100 for testing. Just wondering why there is no validation set (or no cross validation) set. 

2. This is related to 1. The training is performed over 500 epochs but i think, the model is not updated and test_loss is not monitored to stop training early. how is  the best model chosen and saved as pth file. 

3. The UNet model returns the 2 outputs: seg_output, cls_output. Does this mean that the deep learning model performs 2 classification tasks at once ? Namely, the segmentation (P,QRS,T) and the AFIB/AFL classification ?
